src/extension/extensionWeb.ts

```diff
@@ -36,7 +36,7 @@ export async function activate(context: vscode.ExtensionContext) {
     editorEnvelopeLocator: new EditorEnvelopeLocator("vscode", [
       new EnvelopeMapping({
         type: "kaoto",
-        filePathGlob: "**/*.kaoto.+(yml|yaml)",
+        filePathGlob: "**/**(.+(kaoto|camel|kamelet)).+(yml|yaml)",
```

Comment:
might be a good idea to factorize with a constant with the valeu used in extensions.ts

Reply:
If possible, can we make a super simple test that shows what strings are matched with this regexp?

Reply:
this would depend on the regexp implementation used by the multiplying architecture which is not part of this codebase.

It is indirectly tested via UI tests that are checking that the files are opened with kaoto editor or not

Reply:
> might be a good idea to factorize with a constant with the valeu used in extensions.ts

cannot be as part of extension.ts, so I have provided in separate file